src/v2/guide/transitions.md

```diff
@@ -598,7 +598,8 @@ En estos casos, puede también usar el atributo `key` para realizar una trancisi
 </transition>
 ```
 
-Es posible realizar una transición entre cualquier número de elementos, ya sea usando muchos `v-if`, o asignando una propiedad dinámica a un elemento. Por ejemplo:
+Es posible realizar una 
+ón entre cualquier número de elementos, ya sea usando muchos `v-if`, o asignando una propiedad dinámica a un elemento. Por ejemplo:
```

Comment:
I think there's a mistake here. It looks like you accidentally removed the word `transición` while trying to fix something 🤔